scripts/popup.js

```diff
@@ -4,17 +4,45 @@ function save_now_function(){
 	});
 }
 
+
 function recent_capture_function(){
 	var wb_url = "https://web.archive.org/web/2/";
 	chrome.runtime.sendMessage({message: "openurl", wayback_url: wb_url, method:'recent' }, function(response) {
-	});
+			  
+			  if(!response.status){
+			  	notify("URL not found in wayback archives!");
+			  }
+		  
+});
+}
+
+function notify(msg)
+{
+	chrome.notifications.create(
+								'wayback-notification',{   
+								type: 'basic', 
+								iconUrl: '/images/icon@2x.png', 
+								title: "Message", 
+								message: msg
+								},
+								function(){} 
+							);
 }
 
 function first_capture_function(){
 	var wb_url = "https://web.archive.org/web/0/";
 	chrome.runtime.sendMessage({message: "openurl", wayback_url: wb_url, method:'first' }, function(response) {
+	if(!response.status){
+			notify("URL not found in wayback archives!");
+		}	
 	});
 }
+function view_all_function(){
```

Comment:
@rgpt you have done a great job with the visual response. There is just this small glitch which I would like to point out. You have unintendedly created a duplicate function of `view_all_function()`.